### PR TITLE
GRF-540: Disable Enrich Category feature for the 7.0 release

### PR DESCRIPTION
**Pull request by etiennekerbastard**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: config/packages/akeneo_feature_flag.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ akeneo_feature_flag:
         - { feature: 'app_developer_mode', service: 'akeneo_connectivity.connection.app_developer_mode.feature' }
         - { feature: 'reference_data', service: 'akeneo.feature_flag.service.only_flexibility_on_premise_feature' }
         - { feature: 'job_automation_local_storage', service: 'akeneo.feature_flag.service.only_flexibility_on_premise_feature' }
-        - { feature: 'enriched_category', service: 'akeneo.category.enriched.feature'}
+        - { feature: 'enriched_category', service: 'akeneo.feature_flag.service.only_saas_feature'}
```
Comment (BryanS29-akeneo) [inline config/packages/akeneo_feature_flag.yml:12]: replace by the existing service `only_serenity_feature`

File: .env [hunk 2]
```diff
@@ -40,4 +40,3 @@ FLAG_MARKETPLACE_ACTIVATE_ENABLED=1
 FLAG_MARKETPLACE_FAKE_APPS_ENABLED=0
 FLAG_CONNECT_APP_WITH_PERMISSIONS_ENABLED=0
 FLAG_APP_DEVELOPER_MODE_ENABLED=0
-FLAG_ENRICHED_CATEGORY_ENABLED=0
```
Comment (BryanS29-akeneo) [inline .env:43]: do not remove this line

File: src/Akeneo/Category/back/Infrastructure/Symfony/Resources/config/feature_flags.yml [hunk 3]
```diff
@@ -1,9 +0,0 @@
-parameters:
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Category/back/Infrastructure/Symfony/Resources/config/feature_flags.yml:1]: do not remove this declaration of service

File: src/Akeneo/Platform/Bundle/FeatureFlagBundle/Configuration/OnlySaasFeatureFlag.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace Akeneo\Platform\Bundle\FeatureFlagBundle\Configuration;
+
+use Akeneo\Platform\Bundle\FeatureFlagBundle\FeatureFlag;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class OnlySaasFeatureFlag implements FeatureFlag
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Platform/Bundle/FeatureFlagBundle/Configuration/OnlySaasFeatureFlag.php:13]: not necessary to declare an other feature flag service, `OnlySerenityFeatureFlag` already exists

File: src/Akeneo/Platform/Bundle/FeatureFlagBundle/Resources/config/services.yml [hunk 5]
```diff
@@ -49,3 +49,9 @@ services:
         arguments:
             - '%env(string:PIM_EDITION)%'
         public: true
+
+    akeneo.feature_flag.service.only_saas_feature:
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Platform/Bundle/FeatureFlagBundle/Resources/config/services.yml:53]: not necessary

**Review by BryanS29-akeneo — CHANGES_REQUESTED** [review]

**Review by antoinetrouve — APPROVED** [review]

**Review by amigaire — APPROVED** [review]

**Review by BryanS29-akeneo — APPROVED** [review]

**Pull request merged**